### PR TITLE
Add continue buttons to register flow pages

### LINKED ISSUE #93: Add continue buttons in register flow

### PR DESCRIPTION
**Pull request by fremartini**
* each section of the register flow has a continue button that acts in accordance with already defined behavior regarding errors
* rounded_button now hides splash effect when disabled

closes #93

### CODE REVIEW

File: lib/widgets/components/entry/register/register_continue_button.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import 'package:coffeecard/widgets/components/rounded_button.dart';
+import 'package:flutter/material.dart';
+
+class RegisterContinueButton extends StatefulWidget {
```
Comment (TTA777) [inline lib/widgets/components/entry/register/register_continue_button.dart:4]: I don't fully understand the need of the button being a stateful widget. Could you elaborate why this is necessary, or if it isn't make it stateless instead?

**Review by TTA777 — CHANGES_REQUESTED** [review]
One comment I would like addressed. Rest likes fine in my eyes

File: lib/widgets/components/entry/register/register_continue_button.dart (hunk 1 above)
Comment (fremartini) [reply lib/widgets/components/entry/register/register_continue_button.dart:4]: its not, i forgot to remove it :) fixing

**Review by TTA777 — APPROVED** [review]
LGTM, let's get this merged in. If someone else wants to fine-tune the appearance, we can always do so later

**Pull request merged**